lib/chatBot.js

```diff
@@ -990,53 +1030,54 @@ ChatBot.prototype._startWebServer = function(port){
 	var that = this;
 	this.Express = Express;
 	this.express = Express();
-	this.winston.debug("Initializing webserver");
+	this.winston.debug(this.name+"/chatBot.js: Initializing webserver");
 	this.server = http.createServer(this.express);
 	try {
-		this.server.listen(port,function(){that.winston.info('Webserver listening on '+port);});
-		this.winston.debug("Webserver initialized successfully!")
+		this.server.listen(port,function(){that.winston.info(that.name+'/chatBot.js: Webserver listening on '+port);});
 	} catch(err) {
-		this.winston.error(err)
-		this.winston.error("ERROR starting the webserver. Most likely the port is already in use.")
+		this.winston.error(this.name+"/chatBot.js: ERROR starting the webserver. Most likely the port is already in use.",err)
 	}
 	this.express.use(expressWinston.logger({
 		winstonInstance: that.winston,
-		msg: "HTTP {{res.statusCode}} {{(req.header('x-forwarded-for') "+
+		msg: that.name+"/chatBot.js: HTTP {{res.statusCode}} {{(req.header('x-forwarded-for') "+
 		"? req.header('x-forwarded-for').split(',')[0] " +
 		": req.connection.remoteAddress )}} {{req.method}} {{res.responseTime}}ms {{req.hostname+req.originalUrl}}", // customize the default logging message.
 		meta: (that.options.httpMetaLog===false?false:true), // optional: control whether you want to log the meta data about the request (default to true)
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/node-steam-chat-bot/steam-chat-bot/file/969545161/issues/source?bid=1980036&fileBranchId=2848073#l1046)